lib/brewdler/brew_dumper.rb

```diff
@@ -0,0 +1,42 @@
+require "json"
+require "version"
+
+module Brewdler
+  class BrewDumper
+    attr_reader :formulae
+
+    def initialize
+      if system "brew --prefix &>/dev/null"
+        @formulae = JSON.load(`brew info --json=v1 --installed`).map(&method(:formula_inspector))
+      else
+        raise "Unable to list installed formulae. Homebrew is not currently installed on your system."
+      end
+    end
+
+    def to_s
+      @formulae.map do |f|
+        if f[:args].empty?
+          "brew '#{f[:name]}'"
+        else
+          "brew '#{f[:name]}', args: [#{f[:args].map { |arg| "'#{arg}'" }.join(", ")}]"
```

Comment:
Nice work but I think we should pull the `args` stuff onto separate lines to make them more readable.